doc/api/net.md

```diff
@@ -0,0 +1,17 @@
+# net
+
+* [connect(app, client, ...options, \[callback\])](#connectapp-client-options-callback)
+
+## connect(app, client, ...options, \[callback\])
+
+* app: `String || Object` — application to connect to as `'name'` or
+  `'name@version'` or `{ name, version }`, where version must be
+  a valid semver range.
+* client: [`Client`](./client.md#object-client) — JSTP client instance, used
+  only for client—side parts of connections (optional, but either server or
+  client is required).
+* options: `Object` — will be destructured and passed directly to connFactory.
+* callback(error, connection): — Optional callback that will be called when
```

Comment:
You have a dash right next to a colon here, looks like the argument type provided in other cases is missing.